src/schema.rs

```diff
@@ -40,18 +69,13 @@ table! {
     }
 }
 
-table! {
-    messages_day_stat (id) {
-        id -> Integer,
-        date -> Nullable<Date>,
-        userid -> Nullable<Varchar>,
-        message_count -> Nullable<Integer>,
-    }
-}
```

Comment:
where did message_day_stat disappear?

Reply:
Further up due to diesel reorganizing the schema

Reply:
Oh, my bad, didn't see it there xD